feature/settings/import/src/main/kotlin/app/k9mail/feature/settings/import/ui/SettingsImportUiModel.kt

```diff
@@ -11,6 +11,9 @@ internal class SettingsImportUiModel {
     var closeButtonLabel: CloseButtonLabel = CloseButtonLabel.OK
     var isPickDocumentButtonVisible = true
     var isPickDocumentButtonEnabled = true
+    var isPickAppButtonVisible = true
```

Comment:
This ViewModel contains quite some state management. I would consider backing that up by tests. In this state this is very hard to review and check for correctness.